.github/workflows/tests.yml

```diff
@@ -0,0 +1,84 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+      - mampf-next
+      - production
+      - experimental
+  pull_request:
+
+jobs:
+  unit-test-job:
+    name: Execute unit tests & upload to Codecov
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Pull docker images
+      run: docker compose pull --ignore-buildable
+      working-directory: docker/run_cypress_tests
```

Comment:
Maybe rename the folder to `docker/run_tests` as it is not specifically for Cypress tests anymore.